packages/pn-personafisica-webapp/src/components/Notifications/__test__/FilterNotifications.test.tsx

```diff
@@ -189,7 +189,7 @@ describe('Filter Notifications Table Component', () => {
     todayM.setHours(0, 0, 0, 0);
     nineYearsAgo.setHours(0, 0, 0, 0);
     // wrong id and wrong start date
-    await setFormValues(form!, nineYearsAgo, getNextDay(todayM), '1234-5678-910A-BCDFGH-I-OL');
+    await setFormValues(form!, nineYearsAgo, getEndOfDay(todayM), '1234-5678-910A-BCDFGH-I-OL');
```

Comment:
When we click on submit, we transform the dates using getEndOfDay and getStartOfDay.
So we don't need to populate the form with the value already transformed.
Remove the usage of getEndOfDay from this file